e2e/test/scenarios/question-reproductions/reproductions-3.cy.spec.js

```diff
@@ -2610,3 +2614,55 @@ describe("issue 32499", () => {
     }
   });
 });
+
+describe("issue 23449", () => {
+  beforeEach(() => {
+    H.restore();
+    cy.signInAsAdmin();
+  });
+
+  it("Remapped fields should work in a model", () => {
+    // set the metadata for review
+    cy.visit(
+      `/admin/datamodel/database/${SAMPLE_DB_ID}/schema/${SAMPLE_DB_SCHEMA_ID}/table/${SAMPLE_DATABASE.REVIEWS_ID}`,
+    );
+
+    cy.findByTestId("column-RATING").findByLabelText("Field settings").click();
+    cy.findAllByTestId("select-button").contains("Use original value").click();
+    cy.findByLabelText("Custom mapping").click();
+
+    const values = { 1: "A", 2: "B", 3: "C", 4: "D", 5: "E" };
+    for (const [key, value] of Object.entries(values)) {
+      cy.findByDisplayValue(key).clear().type(value);
+    }
+    cy.button("Save").click();
+
+    // make a model on Reviews
```

Comment:
These comments could be `cy.log` statements, to make debugging broken tests easier:

```suggestion
    cy.log("make a model on Reviews")
```

Same goes for the comments below.